services/inc/include.mk

```diff
@@ -2,6 +2,9 @@
 TARGET_SPARK_SERVICES_PATH=$(SERVICES_MODULE_PATH)
 
 INCLUDE_DIRS += $(TARGET_SPARK_SERVICES_PATH)/inc
+INCLUDE_DIRS += $(PROJECT_ROOT)/hal/src/$(PLATFORM_DYNALIB_MODULES)
```

Comment:
Let's better use same pattern as below for nanopb:
```
include $(PROJECT_ROOT)/hal/import.mk
```

Reply:
If I am including this in the makefile itself, I am running into multiple definition problem - 
```
../../../build/arm/startup/startup_rtl872x.S: Assembler messages:
../../../build/arm/startup/startup_rtl872x.S:160: Warning: ignoring changed section attributes for .text
/usr/local/gcc-arm/bin/../lib/gcc/arm-none-eabi/10.2.1/../../../../arm-none-eabi/bin/ld: ../../../build/target/lwip/platform-32-m//liblwip.a(init.o): in function `lwip_init':
/Users/keerthyamisagadda/code/device-os/third_party/lwip/lwip/src/core/init.c:333: multiple definition of `lwip_init'; ../../../build/target/lwip/platform-32-m//liblwip.a(init.o):/Users/keerthyamisagadda/code/device-os/third_party/lwip/lwip/src/core/init.c:333: first defined here
/usr/local/gcc-arm/bin/../lib/gcc/arm-none-eabi/10.2.1/../../../../arm-none-eabi/bin/ld: ../../../build/target/lwip/platform-32-m//liblwip.a(def.o): in function `lwip_htons':
/Users/keerthyamisagadda/code/device-os/third_party/lwip/lwip/src/core/def.c:78: multiple definition of `lwip_htons'; ../../../build/target/lwip/platform-32-m//liblwip.a(def.o):/Users/keerthyamisagadda/code/device-os/third_party/lwip/lwip/src/core/def.c:78: first defined here
/usr/local/gcc-arm/bin/../lib/gcc/arm-none-eabi/10.2.1/../../../../arm-none-eabi/bin/ld: ../../../build/target/lwip/platform-32-m//liblwip.a(def.o): in function `lwip_htonl':
/Users/keerthyamisagadda/code/device-os/third_party/lwip/lwip/src/core/def.c:93: multiple definition of `lwip_htonl'; ../../../build/target/lwip/platform-32-m//liblwip.a(def.o):/Users/keerthyamisagadda/code/device-os/third_party/lwip/lwip/src/core/def.c:93: first defined here
/usr/local/gcc-arm/bin/../lib/gcc/arm-none-eabi/10.2.1/../../../../arm-none-eabi/bin/ld: ../../../build/target/lwip/platform-32-m//liblwip.a(def.o): in function `lwip_strnstr':

```